### PR TITLE
Update CHANGES

### PR DESCRIPTION
**Pull request by adleong**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 * Turn off HTTP decompression so that linkerd doesn't decompress and then
   recompress bodies.
+* Varius bug fixes in the dtab UI
```
Comment (wmorgan) [inline CHANGES.md]: Various (unless that was a @mariusae homage)

File: CHANGES.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 * Turn off HTTP decompression so that linkerd doesn't decompress and then
```
Comment (wmorgan) [inline CHANGES.md]: Streaming bugfix should be in here too

**Comment by olix0r** [comment]
lgtm

File: CHANGES.md (hunk 2 above)
Comment (wmorgan) [inline CHANGES.md]: Also header changes--especially if they are potentially breaking.

**Pull request merged**